applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -56,22 +47,15 @@ def test_Naca0012SmallAdjoint(self):
 
         with WorkFolderScope(work_folder):
             self._runTest(settings_file_name_primal)
+            self._check_results(self.main_model_part.ProcessInfo[CPFApp.LIFT_COEFFICIENT], 0.327805503865, 1e-12, 1e-9)
```

Comment:
Since`abs(0.327805503865) * 1e-12 < 1e-9` the relative tolerance has no effect. In that case it would be more clear to simply pass `0.` instread of `1e-12`.